shoes.gemspec

```diff
@@ -13,7 +13,15 @@ Gem::Specification.new do |s|
   s.description = 'Shoes is the best little GUI toolkit for Ruby. Shoes makes building for Mac, Windows, and Linux super simple. Shoes runs on JRuby only for now.'
   s.license     = 'MIT'
 
-  s.files = Dir["LICENSE", "README.md", "lib/**/*", "ext/install/**/*"]
+  s.files = Dir[
+    "LICENSE", "README.md",
+    "lib/**/*",
+    "ext/install/**/*",
+    "samples/*",
+    "samples/lib/*",
+    "samples/potato_chopping/*",
+    "samples/sounds/*",
```

Comment:
huh, not too sure about packaging up those sounds, that's almost 4 MB. I mean, not huge but still seems a bit excessive starting at our nice 12.5kb

Reply:
Yeah, that's why I trimmed one of the looooong sounds down. Actually I've got an idea how we could easily exclude the sample that uses these from being packaged up but leave it for testing... stay tuned!